<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>191</width>
      <height>791</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>189</width>
       <height>789</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>1211</width>
      <height>791</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>941</width>
        <height>641</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="PlotBackGround"/>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>20</width>
        <height>751</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>30 Day Moving Average</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>60 Day Moving Average</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>200 Day Moving Average</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Stock</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Performance Viewer</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>181</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Plot</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>270</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>to</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Benchmark</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>270</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>101</y>
        <width>1081</width>
        <height>621</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Transactions</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTransactions">
    <property name="title">
     <string>Transactions</string>
    </property>
    <widget class="QMenu" name="menuAdd_Transaction">
     <property name="title">
      <string>Add Transaction</string>
     </property>
     <addaction name="actionSecurity"/>
     <addaction name="actionCash"/>
    </widget>
    <addaction name="actionShow_Transactions"/>
    <addaction name="menuAdd_Transaction"/>
   </widget>
   <widget class="QMenu" name="menuPerformance">
    <property name="title">
     <string>Performance</string>
    </property>
    <addaction name="actionShow_Performance"/>
   </widget>
   <addaction name="menuTransactions"/>
   <addaction name="menuPerformance"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionShow_Transactions">
   <property name="text">
    <string>Show Transactions</string>
   </property>
  </action>
  <action name="actionSecurity">
   <property name="text">
    <string>Security</string>
   </property>
  </action>
  <action name="actionCash">
   <property name="text">
    <string>Cash</string>
   </property>
  </action>
  <action name="actionShow_Performance">
   <property name="text">
    <string>Show Performance</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
